README.md

```diff
@@ -50,23 +55,6 @@ To run postgresql as a service:
 brew services start postgresql
 ```
 
-#### Installing via docker
```

Comment:
I decided to get rid of this because our docker-compose.yml is going to set up containers for the app and sass services as well as the database service. So, this is now redundant.